test/edu/umass/cs/gnsclient/client/singletests/ClientLNSTest.java

```diff
@@ -50,7 +52,9 @@ public ClientLNSTest() {
     if (clientCommands == null) {
       try {
         clientCommands = new GNSClientCommands();
-        clientCommands.setGNSProxy(new InetSocketAddress("127.0.0.1", 24598));
+        //PaxosConfig.getActives() works here because the server and client use the same properties file.
+        InetAddress lnsAddress = PaxosConfig.getActives().values().iterator().next().getAddress();
+        clientCommands.setGNSProxy(new InetSocketAddress(lnsAddress, 24598));
```

Comment:
Is port 24598 never going to change? 
Minor: Good to have a common variable for this port in case it changes.